src/dialogs/zha-device-zigbee-signature-dialog/dialog-zha-device-zigbee-info.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  LitElement,
+  html,
+  css,
+  CSSResult,
+  TemplateResult,
+  customElement,
+  property,
+  query,
+} from "lit-element";
+import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
+import "../../components/dialog/ha-paper-dialog";
+import "../../components/ha-code-editor";
+// Not duplicate, is for typing
+// tslint:disable-next-line
+import { HaPaperDialog } from "../../components/dialog/ha-paper-dialog";
+
+import { PolymerChangedEvent } from "../../polymer-types";
+import { haStyleDialog } from "../../resources/styles";
+import { HomeAssistant } from "../../types";
+import { ZHADeviceZigbeeInfoDialogParams } from "./show-dialog-zha-device-zigbee-info";
+// tslint:disable-next-line: no-duplicate-imports
+import { HaCodeEditor } from "../../components/ha-code-editor";
+import { afterNextRender } from "../../common/util/render-status";
+import { fireEvent } from "../../common/dom/fire_event";
+
+@customElement("dialog-zha-device-zigbee-info")
+class DialogZHADeviceZigbeeInfo extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() private _params?: ZHADeviceZigbeeInfoDialogParams;
+  @property() private _error?: string;
+  @property() private _signature: any;
+  @query("ha-code-editor") private _codeEditor?: HaCodeEditor;
+  @query("ha-paper-dialog") private _dialog!: HaPaperDialog;
+
+  public async showDialog(
+    params: ZHADeviceZigbeeInfoDialogParams
+  ): Promise<void> {
+    this._params = params;
+    const signature = JSON.parse(JSON.stringify(this._params.device.signature));
```

Comment:
Why first stringify and then parse?

Reply:
deep copy, is there a better way?

Reply:
Why do you need a deep copy? You are not going to change it? (and yes, see <https://www.npmjs.com/package/deep-clone-simple>)

Reply:
um i change it on the next line...